packages/search/addon/components/navi-search-result/report.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import Component from '@glimmer/component';
+
+export default class Report extends Component {
+  /**
+   * @property {String} title
+   * @description Title of search result
+   */
+  title = 'Reports & Dashboards';
```

Comment:
So will dashboards be included in report search results? Maybe this shouldn't be called the Report search result then. I think we decided a while ago to call the base class for a report/dashboard an `asset`, although we refer to it as `item` in some other places.

Reply:
I think calling it `Report` is more descriptive that `Asset` or `Item`. It's supposed to show just a list of all the results that are fetched. 
<img width="690" alt="Screen Shot 2020-02-19 at 2 27 45 PM" src="https://user-images.githubusercontent.com/1649445/74873031-0077ca00-5324-11ea-8744-b73bd9f00cd0.png">

Reply:
Report references an actual model though. This search result returns dashboards as well as reports, not just reports so the name of reports is misleading. If you want to have it include both reports and dashboards, it should have a different name as we may need a search result that only returns reports where the name `navi-search-result/report` would be more appropriate.